packages/react-ui/internal/Menu/Menu.styles.ts

```diff
@@ -9,6 +9,7 @@ export const styles = memoizeStyle({
       box-sizing: content-box;
       overflow: auto;
       padding: 0 ${t.menuPaddingX};
+      margin: ${t.menuOffsetY} 0 ${t.menuOffsetY};
```

Comment:
Можно сократить немного
```suggestion
      margin: ${t.menuOffsetY} 0;
```